routes/reviewRoutes.ts

```diff
@@ -0,0 +1,30 @@
+import express from "express";
+import bodyParser from "body-parser";
+import ReviewModel from "../model/review.js";
+import {createNewReview} from "../utils/reviewUtils.js";
+import {checkToken} from "../utils/defaultUtils.js";
+
+const router = express.Router();
+const jsonParser = bodyParser.json()
+
+router.get('/', jsonParser, async (req, res) => {
```

Comment:
What if we want to add finding the reviews of an user? we should have the route as 'product/:productId', so we can make another getter for 'user/:userId' if it's the case